docs/File.md

```diff
@@ -0,0 +1,149 @@
+# File
+
+In this tutorial, we will be building a console application that does basic file logging.
+
+---
```

Comment:
Same comment as above. This may be a personal pref, but I wish there weren't hrs and the steps were numbered. Feels more like a tutorial this way.